java/core/src/main/java/co/worklytics/psoxy/storage/impl/ColumnarBulkDataSanitizerImpl.java

```diff
@@ -76,8 +73,9 @@ public ColumnarBulkDataSanitizerImpl(@Assisted ColumnarRules rules) {
     }
 
     @Override
-    public byte[] sanitize(@NonNull InputStreamReader reader,
-                           @NonNull Pseudonymizer pseudonymizer) throws IOException {
+    public void sanitize(@NonNull InputStreamReader reader,
```

Comment:
> For top efficiency, consider wrapping an InputStreamReader within a BufferedReader. For example:
> BufferedReader in  = new BufferedReader(new InputStreamReader(System.in));
https://docs.oracle.com/javase/8/docs/api/java/io/InputStreamReader.html

```suggestion
    public void sanitize(@NonNull Reader reader,
```
consider leaving implementation to caller?

Reply:
InputStreamReader still an interface ... but yeah, could be more generic